src/main/java/jek/collections/IntArrayList.java

```diff
@@ -20,6 +20,7 @@ public int size() {
 
     @Override
     public boolean isEmpty() {
+
```

Comment:
keep an eye on what you commit. If you use Idea plugin should be easy to recheck all the changes before committing them.

Reply:
committed with git only. next time will try with Idea plugin.